### ISSUE TITLE
Векторные слои (их стили) не отображаются на веб карте, а в QGIS отображаются

### ISSUE THREAD

**Issue by ABiatov** [nextgis/nextgis.com-webgis#29]
Векторные слои (стили):
http://gis-naturalist.nextgis.com/resource/155
http://gis-naturalist.nextgis.com/resource/151
http://gis-naturalist.nextgis.com/resource/160

добавлены на веб-карту http://gis-naturalist.nextgis.com/resource/180 , но их содержимое не отображается на карте, а в таблице атрибутов записи есть.

Эти слои загруженны в nextgiscom через nextgis connect. У каждого слоя созданны стили. При добавлении стилей в QGIS через nextgis connect символика слоев сохраняется.

**Comment by simgislab** [comment]
тип легенды: уникальное значение?

приложи пожалуйста стили в виде QML, есть идея почему не показываются

**Comment by ABiatov** [comment]
В одном из слоев - да, в двух других - нет.
Зато в них сивол двухслойный, вероятно из-за этого.
В приложении зип-архив
[gml.zip.txt](https://github.com/nextgis/nextgis.com-webgis/files/613825/gml.zip.txt)

**Comment by ABiatov** [comment]
Для одного из слоев зоздал новый однослойный простой стиль и добавил его на карту. Слой на карте так и не отобразился.
[Beaver_burrow-simple-style.qml.txt](https://github.com/nextgis/nextgis.com-webgis/files/613836/Beaver_burrow-simple-style.qml.txt)

**Comment by drnextgis** [comment]
Открываю карту - все слои отрисовываются.

**Comment by ABiatov** [comment]
На той карте которая идет в первоначальном посте ( http://gis-naturalist.nextgis.com/resource/162 ) я добавил слои со стилями Mapserver, и они отображаются.
Создал новую карту и добавил на нее слои со стилями созданными в nextgis connect, проблема сохранилась, слои не отображаются:
[http://gis-naturalist.nextgis.com/resource/180](http://gis-naturalist.nextgis.com/resource/180)

**Comment by simgislab** [comment]
а если добавить дефолтные стили в вебе и добавить на ту же карту, покажутся данные?

**Comment by ABiatov** [comment]
Да. Покажутся: [ссылка](http://gis-naturalist.nextgis.com/resource/180/display?base=osm-mapnik&lon=35.2085&lat=50.0905&angle=0&zoom=12&styles=182,181,183,155,151,160)

И даже если не дефолтные, а более слождные созданные как стили мапсервер покажутся: [ссылка](http://gis-naturalist.nextgis.com/resource/162/display?base=osm-mapnik&lon=35.2038&lat=50.0579&angle=0&zoom=15&styles=166,165,173)

**Comment by drnextgis** [comment]
Не похоже, что проблема со стилями. [Вот](http://rykovd.nextgis.com/resource/256/display) я скачал ваши данные и стили и загрузил себе (через веб интерфейс), как видно - всё работает. Может connect что-то не то делает, попробуйте через веб интерфейс загрузить исходные данные, не помогает?
